types/tx.go

```diff
@@ -1,3 +1,24 @@
 package types
 
+import (
+	"github.com/tendermint/go-merkle"
+)
+
 type Tx []byte
+
+type Txs []Tx
+
+func (txs Txs) Hash() []byte {
```

Comment:
the performance impact comes from the garbage collector overhead - it was showing up in the profiling